docs/en/guides/sre/keeper/index.md

```diff
@@ -357,6 +358,25 @@ multi_read  1
 check_not_exists    0
 ```
 
+### HTTP Control {#http-control}
+
+ClickHouse Keeper provides an HTTP interface to check if replica is ready to receive traffic. It may be used for running in cloud environments, e.g [Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes).
```

Comment:
```suggestion
ClickHouse Keeper provides an HTTP interface to check if replica is ready to receive traffic. It may be used for running in cloud environments, such as [Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes).
```